tasks/install_shell.sh

```diff
@@ -176,6 +176,7 @@ if [ -f "$PT__installdir/facts/tasks/bash.sh" ]; then
       "10.12") platform_version="10.12";;
       "10.13") platform_version="10.13";;
       "10.14") platform_version="10.14";;
+      "10.15") platform_version="10.15";;
```

Comment:
Maybe check if `major_version` matches `10\.1[1-9]` (to be on the safe side at least), then pass it on as is? We would get an uglier error when curl-ing the package but the file wouldn't need to be changed for each macOS release.